### ISSUE TITLE
Crashes when pressed shareinfo.

### ISSUE THREAD

**Issue by alian2plus** [SerenaKit/Antoine#7]
Here is the log:

https://pastebin.com/raw/uUEQZD3A

**Comment by plia7** [comment]
@NSAntoine @alian2plus Yes, I can confirm it crashes when clicking on share info. I'm using iPhone 12 Pro Max on iOS 14.4.1 with Taurine jailbreak and rootful version 1.3 of the tweak.

**Comment by brian6932** [comment]
Can reproduce on iOS 15.8.2 palera1n (rootful)
